build-system/pr-check/build.js

```diff
@@ -63,6 +63,7 @@ function main() {
       buildTargets.has('RUNTIME') ||
       buildTargets.has('FLAG_CONFIG') ||
       buildTargets.has('INTEGRATION_TEST') ||
+      buildTargets.has('VISUAL_DIFF') ||
       buildTargets.has('E2E_TEST')
```

Comment:
Remove `E2E_TEST` and `VISUAL_DIFF` from this list since neither of them use `gulp build`.